packages/tasks/src/hardware.ts

```diff
@@ -108,6 +108,14 @@ export const SKUS = {
 				tflops: 30.6,
 				memory: [12, 10],
 			},
+			"RTX 3060 Ti": {
+				tflops: 16.2,
+				memory: [8],
+			},
+			"RTX 3060": {
+				tflops: 12.74,
+				memory: [12],
```

Comment:
```suggestion
				memory: [12, 8],
```

I think there's also a version with 8 GB: https://www.techpowerup.com/gpu-specs/geforce-rtx-3060-8-gb.c3937